net-dns/dnsviz/dnsviz-0.8.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1
+
+MY_PN="dnsviz"
```

Comment:
Umm, maybe I'm not seeing it, but it doesn't look different from `${PN}`?

Reply:
You are right juippis.I removed that one.